researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/ResearchStackDAO.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.researchstack;
+
+import android.content.Context;
+
+import org.researchstack.backbone.model.User;
+import org.sagebionetworks.bridge.android.manager.dao.SharedPreferencesJsonDAO;
+
+/**
+ * Created by jyliu on 2/9/2017.
+ */
+
+public class ResearchStackDAO extends SharedPreferencesJsonDAO {
```

Comment:
Yeah, this is confusing to me also.  We have model classes defined by the rest-client and model classes defined in RS.  It seems like we need a translation between the two?